compiler/statements.go

```diff
@@ -211,10 +211,15 @@ func (fc *funcContext) translateStmt(stmt ast.Stmt, label *types.Label) {
 			iVar := fc.newVariable("_i")
 			fc.Printf("%s = 0;", iVar)
 			keysVar := fc.newVariable("_keys")
-			fc.Printf("%s = $keys(%s);", keysVar, refVar)
```

Comment:
That reminds me... Is `$keys` function used anywhere else in the codebase after this change? If not, we should remove it from prelude.

Reply:
I think it is, but I will check

Reply:
No worries, if it's still used we can keep it. Just wanted to make sure it wasn't completely dead code.